opensrp-anc/src/main/java/org/smartregister/anc/library/activity/MainContactActivity.java

```diff
@@ -217,7 +217,7 @@ private void process(String[] mainContactForms) {
                     }
                 }
                 //Make profile always complete on second contact onwards
-                requiredFieldsMap.put(ConstantsUtils.JsonFormUtils.ANC_PROFILE_ENCOUNTER_TYPE, 0);
+               // requiredFieldsMap.put(ConstantsUtils.JsonFormUtils.ANC_PROFILE_ENCOUNTER_TYPE, 0);
```

Comment:
Why are we disabling this? The workflow dictates all the required fields on Profile are filled up during the 1st contact

Reply:
this line was redundant and causing the data of profile to be overwritten as this line is already invoked in processRequiredStepsField method and is called under the same method where this line was present